worlds/hk/Extractor.py

```diff
@@ -251,7 +251,12 @@ def ast_parse(parser, rule_text, truths: typing.Set[str] = frozenset(), falses:
     for effect in effects:
         effect_names.add(effect["Term"])
     effects = {effect["Term"]: effect["Value"] for effect in effects if
-               effect["Term"] != item_data["Name"] and effect["Term"] != "GEO"}
+               effect["Term"] != item_data["Name"] and effect["Term"] not in {"GEO",
```

Comment:
Might want to add maps to this list too. Maybe others can think of a reason why maps should be progression items, but I can't think of any.